Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_PawnKindDefs.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>WWII Soviet Faction (1.0)</modName>
+			</li>
+
+			<!-- ========== Reduce meals and medicine carried by all pawns ========== -->
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/PawnKindDef[@Name="WWII_SovietPawnKind_Base"]/invNutrition</xpath>
+				<value>
+					<invNutrition>1</invNutrition>
+				</value>
+			</li>
+
+			<li Class="PatchOperationRemove">
+				<xpath>Defs/PawnKindDef[@Name="WWII_SovietPawnKind_Base"]/inventoryOptions</xpath>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/PawnKindDef[
+					defName="WWII_Soviet_Grunt" or
+					defName="WWII_Soviet_Assault" or
+					defName="WWII_Soviet_Assault_TierII" or
+					defName="WWII_Soviet_Sniper" or
+					defName="WWII_Soviet_Sniper_TierII" or
+					defName="WWII_Soviet_Grenadier" or
+					defName="WWII_Soviet_Boss"
+				]/inventoryOptions/subOptionsChooseOne/li[thingDef="MedicineIndustrial"]/countRange</xpath>
+				<value>
+					<countRange>
+						<min>1</min>
```

Comment:
Might set the min to 0. Since I'm not sure if visitors/raiders will actually use medicine to heal themselves, each having one probably makes them rather overly-lucrative to kill on raids.

Reply:
Will definitely set to 0.

One of the authors of the original mod has a tendency to stuff pawns from his faction mods full of medicine, drugs and meals with no regard for item bulk, as if they were being sent on a long camping trip. 

I've done my best in my original patches to slim down the pawn inventory to more reasonable levels, but I'm more than happy to trim any extra fat.

Reply:
Fixed in 2bafaa7